patches/chrome-browser-resources-local_discovery-local_discovery.html.patch

```diff
@@ -0,0 +1,34 @@
+diff --git a/chrome/browser/resources/local_discovery/local_discovery.html b/chrome/browser/resources/local_discovery/local_discovery.html
+index 6a699bafec2866046a71e7b203321c7b38c088f8..49b7daf3ebe5a72b8f1a42d31a4def6b7389cdfa 100644
+--- a/chrome/browser/resources/local_discovery/local_discovery.html
++++ b/chrome/browser/resources/local_discovery/local_discovery.html
+@@ -34,9 +34,6 @@
+                    class="inline-login-promo"
+                    hidden>
+                 <span>$i18n{registerNeedLogin}</span>
+-                <a is="action-link" id="register-overlay-login-button">
```

Comment:
Instead of removing please wrap this with `<if expr=_google_chrome>` like the other html patches.
Ditto below.

Reply:
Fixed and rebased.